Sources/Orbit/Components/Switch.swift

```diff
@@ -117,9 +117,7 @@ struct SwitchPreviews: PreviewProvider {
     static var previews: some View {
         PreviewWrapper {
             standalone
-            storybook
-            storybook
-                .preferredColorScheme(.dark)
```

Comment:
Noticed this in a couple of files - don't we want the dark mode previews anymore?

Reply:
It is the same case as with dynamic size - the Xcode 14 preview support makes this obsolete, as we can see any variant combination with few clicks.